planet/clients/subscriptions.py

```diff
@@ -407,3 +407,38 @@ async def get_results_csv(
         async with self._session.stream('GET', url, params=params) as response:
             async for line in response.aiter_lines():
                 yield line
+
+    async def get_summary(
+        self,
+        subscription_id: Optional[str] = None,
```

Comment:
in the client, i can see an argument for breaking out individual sub summary into its own function, so the shape of the returned dict is consistent. it's not particularly hard to work with right now, but if the schemas diverge over time maybe that makes it simpler to understand...

regardless, a single `planet subscriptions summarize` command is definitely convenient!